getting-started/cli/recipes.md

```diff
@@ -0,0 +1,152 @@
+# Amber Generator Recipes
+
+Amber generators can use a recipe to generate your application, and scaffold your
+application with controllers, models and views.  Using a recipe you can get started
+with an amber application that extends or modifies the standard built in generator
+or provides additional features.  For example you might want an API application
+that only renders JSON or use React or AngularJS for views.
+
+## Recipe repository
+
+Visit the [recipe repository](https://github.com/AmberRecipes/recipes) to discover
+what recipes are available. The available recipes are listed in the Contributions.md
+file and you can drill down into each sub folder for more details about each recipe.
+
+# Amber New Command with a recipe
+
+This is the first command that you will run when generating a new Amber application.
+It creates a new Amber application with a default directory structure and configuration
+setting at the path you specify.
+
+An `.amber.yml` configuration file is generated along with the default application directory
+structure and any additional items or modifications provided by the recipe.
+This stores some metadata for the Amber CLI to make use of for generating migrations,
+scaffolding templates, and more.  The recipe name that you use to generate the application is saved
+in the `.amber.yml` file so that when you generate controllers, models and views with the
+scaffold generator the same recipe is used - you don't have to specify the recipe each time.
+
+### `amber new`
+
+```shell
+amber new NAME OPTIONS
+
+Arguments:
+  NAME  name/path of project
+
+Options:
+  -d      Select the database database engine, can be one of: pg | mysql | sqlite (default: pg)
+  --deps  Installs project dependencies, this is the equivalent of running (shards update)
+  -t      Selects the template engine language, can be one of: slang | ecr (default: slang)
+  -r      Use a named recipe.  See documentation at https://amberframework.org.
+```
+
+## Example Usage
+Using `amber new microsecond-blog -r damianham/modular` will generate a skeleton
+Amber application in `./microsecond-blog` using the _modular_ recipe that was
+provided by the github user _damianham_.
+
+You can have a running web application in a matter of minutes:
+1. `amber new microsecond-blog -r damianham/modular`
+1. `cd microsecond-blog`
+1. `shards install`
+1. `amber run &` or `amber watch &` (also can execute these in foreground of separate terminal)
+1. `open http://localhost:3000`
```

Comment:
This doesn't need to be a cli command, and it could even be a clickable link instead.